build.gradle

```diff
@@ -1,6 +1,30 @@
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.2.7.RELEASE'  apply false
+    id "org.springframework.boot" version "2.4.1" apply false
+    id "org.springdoc.openapi-gradle-plugin" version "1.3.0"
+    id "com.github.johnrengelman.processes" version "0.5.0"
+}
+
+allprojects {
+    apply plugin: 'io.spring.dependency-management'
+
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+
+    dependencyManagement {
+        overriddenByDependencies = false
+
+        dependencies {
+            dependency 'org.springdoc:springdoc-openapi-security:1.5.2'
+        }
+    }
+
+    openApi {
+        apiDocsUrl.set("https://localhost:9000/api/docs")
+        outputDir.set(file("$buildDir/docs"))
+    }
```

Comment:
this project itself doesn't produce any apis, so doesn't make sense to expose anything, especially in all projects